common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -519,6 +519,7 @@ def _render_student_view_for_items(self, context, display_items, fragment, view=
         for item in display_items:
             # NOTE (CCB): This seems like a hack, but I don't see a better method of determining the type/category.
             item_type = item.get_icon_class()
+            item_icon = item.get_icon()
```

Comment:
Is `item_type` still used? It doesn't seem right that we call both `item.get_icon_class()` and `item.get_icon()` here, and use both results. I expected that `get_icon` would replace `get_icon_class` here, especially since by default `get_icon` just calls `get_icon_class`.

If we do need to keep both, perhaps rename the first one to `get_css_class()` ?

Reply:
Yep, good point. I kept this for screen readers [here](https://github.com/edx/edx-platform/blob/9f39ca689d8ae5429a9ea77e34db1fde3e1ede44/lms/templates/seq_module.html#L74), but it makes more sense to override this everywhere. I remove the `item_icon` and replaced `get_icon_class` with `get_icon`.